docs/http-challenge/s3.md

```diff
@@ -0,0 +1,72 @@
+# Variables for s3 http-challenge
+
+| Variable                            | Required | Default   | Description
+|-------------------------------------|----------|-----------|------------
+| letsencrypt_s3_bucket_name          | yes      |           | name of the s3 bucket which should be used
+| letsencrypt_s3_access_key           | yes      |           | aws access key for API user of s3 bucket
+| letsencrypt_s3_secret_key           | yes      |           | aws secret key for API user of s3 bucket
+| letsencrypt_s3_config_region        | no       | us-west-1 | aws s3 region in which bucket can be found
+
+## Validation
+
+You have to set a redirect rule in your proxy or webserver to allow the acme challenge bot to read the file, during the http-01 challenge to work:
+
+**HaProxy(Version <=> 1.5):**
```

Comment:
Should this be `<= 1.5`?

Reply:
No, this is correct as this syntax should work for all HaProxy versions. At least this was tested in 1.4 and 1.5. With Version 1.6 a better way has been implemented but also the old variant will work.

Do you think we should change it to some kind of "works for all" header?

Reply:
`Version <=> 1.5` means all versions EXCEPT 1.5.

> Do you think we should change it to some kind of "works for all" header?
Yes.

Reply:
Adjusted a4688f0313cfd95b53da8d888590443e52c4283a